spec/resources/workbooks_spec.rb

```diff
@@ -110,23 +109,55 @@ def find_or_publish_workbook(name)
     end
   end
 
+  describe '#permissions' do
+    it 'can retrieve the permissions of a workbook' do
+      workbook = find_or_publish_workbook('testpublish')
+      expected = [{
+        'granteeType' => 'group',
+        'granteeId' => all_users_group['id'],
+        'capabilities' => {
+          'Read' => true,
+          'ExportData' => true,
+          'ViewComments' => true,
+          'AddComment' => true,
+          'ExportImage' => true
+        }
+      }, {
+        'granteeType' => 'group',
+        'granteeId' => test_group['id'],
+        'capabilities' => {
+          'Read' => true,
+          'ChangePermissions' => false
+        }
+      }, {
+        'granteeType' => 'user',
+        'granteeId' => test_user_id,
+        'capabilities' => {
+          'Read' => true,
+          'ChangePermissions' => false
+        }
+      }]
+      actual = client.workbooks.permissions(workbook_id: workbook['id'])
+      expect(actual).to eq expected
```

Comment:
Nitpicky, but, stylistically, the rest of the tests are `expect(client.workbooks...).to eq expected`. Not sure if it's worth changing.